### PR TITLE
Add archive icon to the icon set

### PR DESCRIPTION
**Pull request by StorytellerCZ**

### CODE REVIEW

File: v3.4/fonts/percolate.svg [hunk 1]
```diff
@@ -121,7 +121,7 @@
...
....476-3.266-7.416-8.522-16.12-14.874zM814.532 445.536v-93.24c-2.474-3.266-7.416-8.522-16.12-14.874-15.6-11.386-37.69-22.346-63.882-31.696-60.984-21.77-140.002-33.758-222.498-33.758s-161.514 11.988-222.498 33.758c-26.192 9.348-48.282 20.308-63.88 31.696-8.706 6.352-13.646 11.608-16.122 14.874v93.24c70.29-37.48 179.654-61.566 302.5-61.566s232.21 24.086 302.5 61.566zM225.652 750.854c15.6 11.386 37.69 22.346 63.88 31.696 60.984 21.77 140.002 33.758 222.498 33.758s161.514-11.988 222.498-33.758c26.192-9.348 48.282-20.308 63.882-31.696 8.704-6.352 13.646-11.608 16.12-14.874v-0.026c-2.474-3.266-7.416-8.522-16.12-14.874-15.6-11.386-37.69-22.346-63.882-31.696-60.984-21.77-140.002-33.758-222.498-33.758s-161.514 11.988-222.498 33.758c-26.192 9.348-48.282 20.308-63.88 31.696-8.706 6.352-13.646 11.608-16.122 14.874v0.026c2.476 3.268 7.418 8.524 16.122 14.874zM798.412 145.422c-15.6-11.386-37.69-22.346-63.882-31.696-60.984-21.77-140.002-33.758-222.498-33.758s-161.514 11.988-222.498 33.758c-26.192 9.348-48.282 20.308-63.88 31.696-8.714 6.36-13.66 11.62-16.13 14.886h0.010v93.228c70.29-37.48 179.654-61.566 302.5-61.566s232.21 24.086 302.5 61.566v-93.228h0.010c-2.474-3.266-7.42-8.526-16.132-14.886z" />
```
Comment (filipenevola) [inline v3.4/fonts/percolate.svg]: Why this change?

Remember that this is used in many places.

**Review by filipenevola — COMMENTED** [review]
All these changes were really necessary to add an icon?

File: v3.4/fonts/percolate.svg (hunk 1 above)
Comment (StorytellerCZ) [reply v3.4/fonts/percolate.svg]: That was automatic change, I have cough it in the css, but missed it here. Fixing.

**Review by filipenevola — APPROVED** [review]
